src/components/Text/Text.stories.tsx

```diff
@@ -1,6 +1,7 @@
 import type { Meta, StoryObj } from "@storybook/react";
 import Text, { textSizesArray } from "./Text";
 import { argsBooleanType } from "../../helpers/storybookUtils";
+import formatNumber from "../../hooks/useFormatNumber";
```

Comment:
@EdwinGuzman I was expecting for this to be wrapped in brackets when it is imported, but it works fine like this. In fact, when I put it in brackets, it throws an error.

Reply:
In the hook file, the function is the default export so this is correct for that export. If we want to use braces, then it additionally needs to export the function as a named function `export useFormatNumber;`.

Either is ok and ultimately it will matter the most when it gets exported in the main `index.ts` file... which actually is needed here in order for consuming apps to use. So @deepikagonuguntla make sure to export this hook in the main index.ts file.